auctionapp/auction-app
# FB OAuth - chris

#### What's this PR do?

Complete FB OAuth, create frameworks for landing page and item sale form
#### Where should the reviewer start?

FB OAuth mainly in client/components/FacebookButton.jsx, client/components/Signin.jsx, server/app.js. Landing page and item sale form in client/components/Landing.jsx and client/components/SellItem.jsx
#### How should this be manually tested?

Click on the login button on the home page. It should prompt you to login with Facebook and then redirect to home page if you confirm
#### Any background context you want to provide?

No
#### What are the relevant tickets?

n/a
